presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add fragment result caching support for ``UNNEST`` queries.
+* Add :func:`poisson_cdf` and :func:`inverse_poisson_cdf` functions.
+* Add memory tracking in ``TableFinishOperator`` which can be enabled by setting the ``table-finish-operator-memory-tracking-enabled`` configuration property to ``true``. Enabling this property can help investigating GC issues on the coordinator, by allowing us to debug whether stats collection uses a lot of memory.
+* Remove spilling strategy ``PER_QUERY_MEMORY_LIMIT`` and instead add configuration property ``experimental.query-limit-spill-enabled`` and session property ``query_limit_spill_enabled``. When ``query_limit_spill_enabled`` is set to ``true`` and the spill strategy is not ``PER_TASK_MEMORY_THRESHOLD``, then we will spill whenever a query uses more than the per-node total memory limit in combined revocable and non-revocable memory.  We will also still spill whenever the memory pool exceeds the spill threshold. This fixes an issue where using the ``PER_QUERY_MEMORY_LIMIT`` spilling strategy could prevent the oom killer from running when the memory pool was full. The issue is still present for the ``PER_TASK_MEMORY_THRESHOLD`` spilling strategy.
+
+Hive Changes
+____________
+* Add support for MaxResults on Glue Hive Metastore. MaxResults is the maximum number of partitions to return in a single response.
+* Add support for bucket sort order in Glue when creating or updating a table or partition.
+* Add support for partition cache validation. This can be enabled by setting the ``hive.partition-cache-validation-percentage`` configuration property. Partition cache validation allows us to validate the value returned from partition cache with the actual value from Metastore. The default value is 0.0 which means there is no validation.
+* Add support for partition schema evolution for parquet.
+* Add support for Glue endpoint URL :doc:`/connector/hive`.
+* Add support for the S3 Intelligent-Tiering storage class for writing data. The S3 storage class can be configured using the configuration property ``hive.s3.storage-class``. Supported values are ``STANDARD`` and ``INTELLIGENT_TIERING``, and the default value is ``STANDARD``.  :doc:`/connector/hive`.
+* Add configuration property ``hive.metastore.glue.max-error-retries`` for the maximum number of retries for glue client connections. The default value is 10.  :doc:`/connector/hive`.
+* Allow accessing tables in Glue metastore that do not have a table type.
```

Comment:
Maybe start this "Add support for" instead of "Allow", so it matches the other release notes.